components/collapsible-panel/collapsible-panel-group.js

```diff
@@ -0,0 +1,69 @@
+import { css, html, LitElement } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { SkeletonMixin } from '../skeleton/skeleton-mixin.js';
+
+/**
+ * A component that renders collapsible panels
+ * @slot default - Slot for panels. Only accepts `d2l-collapsible-panel`
+ */
+class CollapsiblePanelGroup extends SkeletonMixin(LitElement) {
+
+	static get properties() {
+		return {
+			_panels: { state: true },
+		};
+	}
+
+	static get styles() {
+		return [super.styles, css`
+			:host ::slotted(*) {
+				display: none;
+			}
+			:host ::slotted(d2l-collapsible-panel) {
+				display: unset;
+			}
+			.spaced {
+				display: flex;
```

Comment:
I'm not sure if there are any side-effects of overriding the slot's `display: contents`. If not, this is great (and good to know!). If there is, you might need a wrapper div to apply the flex layout.

Reply:
I'm not sure if there is....it looked fine to me in the demo page...but I can wrap in a div for safety

Reply:
Did a bit of research on this -- it _seems_ safe to override the `display: contents`.

From the [CSS spec](https://drafts.csswg.org/css-display/#box-generation):

> The element itself does not generate any boxes, but its children and pseudo-elements still generate [boxes](https://drafts.csswg.org/css-display/#box) and [text sequences](https://drafts.csswg.org/css-display/#css-text-sequence) as normal. For the purposes of box generation and layout, the element must be treated as if it had been replaced in the [element tree](https://drafts.csswg.org/css-display/#element-tree) by its contents


I'm _guessing_ we're safe to change this to `flex`? Since the `slot` is now acting as an element that controls layout (vs. an "invisible" element).